opendbc/car/extra_cars.py

```diff
@@ -28,6 +28,16 @@ def init_make(self, CP: structs.CarParams):
 class CAR(Platforms):
   config: ExtraPlatformConfig
 
+  HONDA_CANFD = ExtraPlatformConfig(
+    [
+      CommunityCarDocs("Acura Integra 2024", "All"),
```

Comment:
Integra isn't actually CANFD, just regular CAN

Reply:
I have some minor updates to add soon, I'll re-categorize Integra and add Clarity while doing that. Thanks!